LayoutTests/imported/w3c/web-platform-tests/trusted-types/modify-attributes-in-callback.html

```diff
@@ -13,11 +13,14 @@
 
   let target = "data-x";
   trustedTypes.createPolicy("default", {
-    createHTML: (s) => {
+    createHTML: (s, _, sink) => {
+    // TODO: sink value?
+    assert_equals(sink, '');
```

Comment:
I think the sink value here per spec should be "HTMLIFrameElement srcdoc". This needs updating in the WebKit implementation AND the Chromium implementation.

Reply:
Thanks. Will check both implementations.

Reply:
I think we might need to add `sink` paramenter in `DOMString? [getAttributeType](https://w3c.github.io/trusted-types/dist/spec/#dom-trustedtypepolicyfactory-getattributetype)(`

Reply:
You are right. I changed the implementation at https://github.com/WebKit/WebKit/pull/26519 so now we have the sink value passed over as "HTMLIFrameElement srcdoc".